R/pkg/R/mllib.R

```diff
@@ -142,15 +143,6 @@ setMethod("spark.glm", signature(data = "SparkDataFrame", formula = "formula"),
 #' Generalized Linear Models (R-compliant)
 #'
 #' Fits a generalized linear model, similarly to R's glm().
-#' @param formula A symbolic description of the model to be fitted. Currently only a few formula
```

Comment:
Is there a reason we are moving these?

As we have discussed, we should try to keep the documentation as close to the function definition/implementation as possible. That would make it a lot more easier to keep the code and documentation in sync, and to remind reviewers to check for them for correctness and consistency.

Where there are cases where a @param description applies to two or more implementations (because it supports multiple classes) because of the lack of a common place we could put that in the generics.R for now.

If `...` only shows up in generics.R - we should check if it's because it's to match an existing function signature, if it is then we add @param in generics.R. If it is not and it is not used anywhere we should consider removing the `...`

Does that make sense?